ShoppingList/Articles.cs

```diff
@@ -3,34 +3,32 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using static System.Console;
 
 namespace ShoppingList
 {
-    public class Articles
+    public class Articles : ExampleList
     {
         public string NameOfThing;
         public int amount;
         public decimal price;
-        private decimal realPrice;
         public Articles() // Construct 
         {
             NameOfThing = "Not Define";
             amount = 0;
-            price = 0;
-            if(amount > 1)
-            {
-                realPrice = this.RealPrice();
-            }
-            else
-            {
-                realPrice = price;
-            }
         }
-        private decimal RealPrice()
+        public void CreateANewArticle()
```

Comment:
This method should not be depended from readLine, it should accept parameters from outside